app/src/main/kotlin/com/wire/android/ui/common/Extentions.kt

```diff
@@ -12,6 +12,7 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.graphics.vector.ImageVector
 import androidx.compose.ui.semantics.Role
+import com.wire.android.ui.theme.wireDimensions
```

Comment:
I'm not sure about it, as `fun folderWithElements()` from `conversationslist.common.Extention.kt` is something specific for the `ConversationsList` only. And `ui.common.Extention.kt` suppose to contain only funs that could be useful anywhere is the project :)

Reply:
If that is so, then maybe we should move it to `package com.wire.android.ui.home.conversationslist` and mark this `fun folderWithElements()` method as `@PackagePrivate` to avoid confusion and indicate it can only used at that level.
Regarding this file, I would still fix the naming to `Extensions` 👓